packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -343,11 +339,8 @@ function createSequenceInterval(
         endRefType = ReferenceType.Transient;
     }
 
-    // TODO: Should SlideOnRemove be the default behavior?
-    if (intervalType & IntervalType.SlideOnRemove) {
-        beginRefType |= ReferenceType.SlideOnRemove;
-        endRefType |= ReferenceType.SlideOnRemove;
-    }
+    beginRefType |= ReferenceType.SlideOnRemove;
```

Comment:
behavior change should be doc'd once you add doc

Reply:
behavior change documented